src/main/resources/python/stack.py

```diff
@@ -4,22 +4,28 @@ def __init__(self, ul, ur, ll, lr, aligned_edge, color=WHITE, text_color=WHITE,
         super().__init__(ul, ur, ll, lr, aligned_edge, color, text_color, text_weight, font)
         self.empty = None
 
-    def create_init(self, text=None):
+    def create_init(self, text=None, creation_style=None):
+        if not creation_style:
+            creation_style = "ShowCreation"
         empty = Init_structure(text, 0, self.max_width - 2 * MED_SMALL_BUFF, color=self.color, text_color=self.text_color)
         self.empty = empty.all
         empty.all.move_to(np.array([self.width_center, self.lr[1], 0]), aligned_edge=self.aligned_edge)
         self.all.add(empty.all)
-        return ShowCreation(empty.all)
+        creation_transform = globals()[creation_style]
+        return [creation_transform(empty.text), ShowCreation(empty.shape)]
```

Comment:
This is really minor but do we want to use the same creation_style passed in for both text and shape or just the text?

Reply:
I tried this, but some of the transforms are made mainly with text in mind (e.g., `Write`) so applying them to the shape looks a bit weird. But I can add a check for those ones maybe?

Reply:
I see. I think it's fine then - probably don't need to add the check since there aren't that many different creation animation other than `ShowCreation`? The only other one I can think of is `FadeIn`, but it's probably not worth the effort to add all the checks in just for that. I'm not sure what do you think?

Reply:
I've actually put a list of the available creation animations that I was able to find [here](https://github.com/ManimDSL/ManimDSLCompiler/blob/820a7de53a1163f627ef24ad705ca6730a66e5d7/src/main/kotlin/com/manimdsl/stylesheet/StyleSheetValidator.kt#L11). Let me know what you think and whether I should add a check? From what I remember, `Write` and `DrawBorderThenFill` look a bit strange when applied to shapes

Reply:
Personally, I think it's fine either way depending on how hard it is to add the check? I think it's fine if we just leave it the way it is, especially if it takes a lot of effort to add in the check, but I'm not sure how much tony cares about this... maybe others can put in their opinion on this.

Reply:
This is just for the init part right? When it comes to pops and pushes it can would be for the whole shape? Or still only the text?

Reply:
Yep, whole shape for pushes. I actually didn't add these animations to pops, because I thought they might look a bit strange considering the fact that we are already moving them up and then fading them out

Reply:
Okay nice, yeah makes sense, I think if it's just the init stack thing we don't need to worry about it

Reply:
The rest looks good I think

Reply:
Okay great - I just need to add tests and then hopefully this should be ready to go :)